lightning-tag/detailed-reference.md

```diff
@@ -1,3 +1,96 @@
-# Detailed Reference
+# Detailed Reference Guide
 
-?> Coming Soon
+!> Under Construction.
+
+## Platform Setup
+
+Before implementing the Lotame Lightning Tag on your site, there is setup needed to be completed inside the Lotame Platform as described in the [Pre-Implementation Setup Tasks](lightning-tag/implementation-setup-tasks.md)
+
+## Javascript Setup
+
+Setup of the Lotame Lightning Tag javascript on your website involves Passing a input object to a function which loads the window namespace on your page which is where the javascript tag loads itself into an executes. The basic setup looks like the below:
```

Comment:
Alternative wording:
To set up the Lotame Lightning Tag javascript on your website, copy the snippet below and adjust the `lotameTaginput` object as detailed in Config Object and Data Object sections .

Reply:
I really don't want people copy/pasting from this page instead of the other guides. This is meant as a reference of all possible public functionality, not a how-to.

Does that make sense?

With that said, I agree that block of text sucks and needs tweaking. 😄

Reply:
Try #2: Setup of the Lotame Lightning Tag javascript on your website involves an input object that gets passed to an initialization function that prepares a window namespace on your page for the javascript tag itself. The basic setup looks like the below:

Reply:
I like it. Done.